packages/flutter/test/widgets/editable_text_shortcuts_test.dart

```diff
@@ -2201,4 +2208,106 @@ void main() {
     }, variant: appleOnly);
 
   }, skip: !kIsWeb);// [intended] specific tests target web.
+
+  group('Web does accept', () {
+    group('desktop platforms', () {
+      const SingleActivator selectUp =
+          SingleActivator(LogicalKeyboardKey.arrowUp, shift: true);
+
+      const SingleActivator selectDown =
+          SingleActivator(LogicalKeyboardKey.arrowDown, shift: true);
```

Comment:
To be consistent with `selectAllUp`/`selectAllDown`, let's move these inside their respective test.